incubator/busbar/values.yaml

```diff
@@ -0,0 +1,48 @@
+## Busbar default Values
+##
+
+## Busbar Image
+image:
+  name: busbario/busbar
+  tag: 1.5.1
+  pullPolicy: Always
+
+## Cluster Name
+clusterName: busbar_cluster
+privateDomainName: private
+publicDomainName: example.com
+
+## Private Registry
+registryStorageS3Accesskey: my_private_registry_s3_access_key
+registryStorageS3Secretkey: my_private_registry_s3_secret_key
+registryStorageS3Bucket: my_private_registry_s3_bucket
+
+kubeRegistry:
+  resources:
```

Comment:
Don't specify defaults for resources. We usually recommend to leave this a conscious choice of the user.